src/ui-config/reservePatches.ts

```diff
@@ -59,6 +61,7 @@ export const SYMBOL_NAME_MAP: { [key: string]: string } = {
   GUSD: 'Gemini Dollar',
   KNC: 'Kyber Legacy',
   LINK: 'ChainLink',
+  MAI: 'MIMATIC',
```

Comment:
On polygon MAI symbol is miMATIC: https://polygonscan.com/address/0xa3Fa99A148fA48D14Ed51d610c367C61876997F1#readContract

Reply:
ok, added symbol fix to `SYMBOL_MAP`